### PR TITLE
chore: bump poetry version

### PR DESCRIPTION
**Pull request by cpcloud**
Bumps the version of poetry.

### CODE REVIEW

File: pkgs/poetry/update.nix [hunk 1]
```diff
@@ -1,6 +1,5 @@
 let
-  sources = import ../../nix/sources.nix;
-  pkgs = import sources.nixpkgs {
+  pkgs = import <nixpkgs> {
```
Comment (cpcloud) [inline pkgs/poetry/update.nix:2]: @adisbladis Is there a better way to do this that doesn't depend on special path syntax?

Comment (cpcloud) [reply pkgs/poetry/update.nix:2]: I will rewrite this as a flake application in a separate PR and then rebase this PR on that one.

**Comment by andronat** [comment]
epic!

**Review by brianmay — APPROVED** [review]

**Comment by cpcloud** [comment]
Merging! If there are any issues we can always revert.

**Pull request merged**